bittensor/utils/btlogging/loggingmachine.py

```diff
@@ -443,45 +443,65 @@ def __trace_on__(self) -> bool:
         """
         return self.current_state_value == "Trace"
 
-    def trace(self, msg="", prefix="", suffix="", *args, **kwargs):
+    def trace(
+        self,
+        msg="",
+        prefix="",
+        suffix="",
+        *args,
+        stacklevel=2  # logger.trace is not an internal method
+        if sys.version_info >= (3, 11)
```

Comment:
There are several options to pass the value here, but not like this:
1. define a constant at the beginning of the module and pass it as a value;
2. pass a default value as None, then determine the value inside the function if it is None.

Reply:
thanks for pointing that out!